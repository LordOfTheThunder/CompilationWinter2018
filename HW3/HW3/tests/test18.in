struct a {
	int t;
};
struct a {
	int z;
};
