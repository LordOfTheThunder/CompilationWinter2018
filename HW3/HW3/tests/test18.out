line 5: identifier struct a is already defined
